libs/features/mas/src/variants/catalog.js

```diff
@@ -80,14 +81,18 @@ export class Catalog extends VariantLayout {
     };
 
     toggleActionMenu = (e) => {
-      const actionMenuContentSlot = this.card.shadowRoot.querySelector(
+      const shadowRoot = this.card.shadowRoot;
+      const actionMenu = shadowRoot.querySelector('.action-menu');
+      const actionMenuContentSlot = shadowRoot.querySelector(
```

Comment:
both actionMenu and actionMenuContentSlot could have been getter?
to avoid duplication in line 84 and 116

Reply:
Fixed

```
get actionMenu() {
    return this.card.shadowRoot.querySelector('.action-menu');
}

get actionMenuContentSlot() {
    return this.card.shadowRoot.querySelector('slot[name="action-menu-content"]');
}
```